core/src/com/unciv/logic/multiplayer/chat/ChatStore.kt

```diff
@@ -0,0 +1,92 @@
+package com.unciv.logic.multiplayer.chat
+
+import com.badlogic.gdx.Gdx
+import com.unciv.logic.event.Event
+import com.unciv.logic.event.EventBus
+
+data class ChatMessageSendRequested(
+    val gameId: String,
+    val civName: String,
+    val message: String,
+) : Event
+
+data class ChatMessageReceived(
+    val gameId: String,
+    val civName: String,
+    val message: String,
+) : Event
+
+data class Chat(
+    val gameId: String,
+) {
+    // <civName, message> pairs
+    private val messages: MutableList<Pair<String, String>> = mutableListOf(INITIAL_MESSAGE)
+
+    val length: Int get() = messages.size
+
+    /**
+     * Only requests a message to be sent, does not guarantee delivery.
+     *
+     * The server will relay it back if a delivery was acknowledged and that is when we should display it.
+     * **/
+    fun requestMessageSend(civName: String, message: String) {
+        EventBus.send(ChatMessageSendRequested(gameId, civName, message))
```

Comment:
Don't use the event bus, it makes things needlessly complicated and should be deprecated

Reply:
Then what should I use.

Reply:
Events should be handled in the chatstore. If you must have an event bus, it should be localized to the class using it, not general purpose - that just causes spaghetti

Reply:
Wow, nice catch! I realized that we don't need this event after all. Fixed by https://github.com/yairm210/Unciv/pull/13599/commits/f03de1e4f3e85efb3c14dce1f66b3e25f7ce9357.